docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -346,58 +384,75 @@ spec:
 - Completely removing the control plane and issuing a delete on the underlying machines.
 - User documentation should focus on deletion of the Cluster resource rather than the KubeadmControlPlane resource.
 
-##### KubeadmControlPlane rollout (using create-swap-and-delete)
+##### KubeadmControlPlane rollout
 
-- Triggered by: 
-    - Changes to Version
-    - Changes to the kubeadmConfigSpec
-    - Changes to the infrastructureRef
-    - The `upgradeAfter` field, which can be set to a specific time in the future
-        - Set to `nil` or the zero value of `time.Time` if no upgrades are desired
-        - An upgrade will run when that timestamp is passed
-        - Good for scheduling upgrades/SLOs
-        - Set `upgradeAfter` to now (in RFC3339 form) if an upgrade is required immediately
-
-- Rollout operations rely on scale up and scale down which are be blocked based on Etcd and control plane health checks
+KubeadmControlPlane rollout operations rely on [scale up](#scale up) and [scale down](#scale_down) which are be blocked based on Etcd and control plane health checks
   - See [Health checks](#Health checks) below.
 
-- The rollout algorithm is the following:
+KubeadmControlPlane rollout is triggered by:
+  - Changes to Version
+  - Changes to the kubeadmConfigSpec
+  - Changes to the infrastructureRef
+  - The `upgradeAfter` field, which can be set to a specific time in the future
+    - Set to `nil` or the zero value of `time.Time` if no upgrades are desired
+    - An upgrade will run when that timestamp is passed
```

Comment:
Unrelated to this PR, but I think it would help to clarify that we don't perform a rollout exactly at the time of upgradeAfter

```suggestion
    - An upgrade will run after that timestamp is passed
```